service/operator/operator.go

```diff
@@ -79,12 +81,29 @@ func New(config Config) (*Operator, error) {
 		c.Name = awstpr.Name
 		c.Version = awstpr.VersionV1
 
-		newTPR, err = tpr.New(c)
+		awsTPR, err = tpr.New(c)
 		if err != nil {
 			return nil, microerror.Maskf(err, "creating TPR util for "+awstpr.Name)
 		}
 	}
 
+	var cfTPR *tpr.TPR
```

Comment:
As above there won't be a Cloud Formation TPR we just need awstpr.

Reply:
Yep, fixed